src/gantry/screens.py

```
@@ -1110,469 +1113,59 @@ def on_search_input_search_changed(self, message: SearchInput.SearchChanged) ->
-            "sidebar": "table",  # Fallback if somehow in sidebar state
-            "table": "table",
-        }
-        next_panel = next_panels.get(self.current_panel, "table")
-        self.current_panel = next_panel
-
-        # Move focus to the target panel widget
-        try:
-            if next_panel == "table":
-                self.query_one("#chart-table", ResourceTable).focus()
+            content = path.read_text(errors="replace")
         except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)